app/controllers/metadata_controller.rb

```diff
@@ -29,5 +29,17 @@ def default_manifest
         }
       ]
     }
+    if SiteSetting.native_app_install_banner
```

Comment:
I prefer to leave a blank line before/after a multi-line code block to make it easier to read.